oracledb-ballerina/tests/02-execute-basic-test.bal

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we follow one  convention in naming test files. Some are ended as `-test.bal` while others are not.

Reply:
Fixed with 28468e1a9bbdfaf47b9d0a048879f548137638ca